examples/experimental/dagster-dlift/dagster_dlift/translator.py

```diff
@@ -95,3 +113,27 @@ def get_spec(self, data: DbtCloudContentData) -> Union[AssetSpec, AssetCheckSpec
             return self.get_test_spec(data)
         else:
             raise Exception(f"Unrecognized content type: {data.content_type}")
+
+
+class DagsterDbtCloudRuntimeTranslator:
```

Comment:
Interesting - this pattern makes sense to me, but I wonder if we should be using the translator name or if we should come up with something else.

I also wonder if it would be useful in terms of reducing complexity to merge this with the other translator, even if the purposes don't align. What do you think?

Reply:
One thing we might want to adjust is all the `translator` param names to `runtime_translator` and `load_translator` if we keep this, because now it's a bit unclear which translator type each accepts

Reply:
Yea I mean you could imagine the translator being used for both cases, it's just an extension of it's current function ultimately. I don't feel super strongly.